backend/src/exceptions/auth-error/auth-error.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpError } from '~/exceptions/exceptions';
+import { CustomExceptionName, HttpCode } from '~/common/enums/enums';
+
+type Constructor = {
+  message?: string;
+  status?: number;
+};
+
+class AuthError extends HttpError {
+  constructor({
+    message,
+    status = HttpCode.INTERNAL_SERVER_ERROR,
```

Comment:
Let's remove default messages for services at all
```suggestion
    status,
```